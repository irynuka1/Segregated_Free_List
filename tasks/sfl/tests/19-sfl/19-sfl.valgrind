==3478== Memcheck, a memory error detector
==3478== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3478== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==3478== Command: /home/irynuka1/Desktop/UNI/SDA/Tema1/sfl
==3478== Parent PID: 3477
==3478== 
==3478== 
==3478== HEAP SUMMARY:
==3478==     in use at exit: 0 bytes in 0 blocks
==3478==   total heap usage: 92 allocs, 92 frees, 9,864 bytes allocated
==3478== 
==3478== All heap blocks were freed -- no leaks are possible
==3478== 
==3478== For lists of detected and suppressed errors, rerun with: -s
==3478== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
