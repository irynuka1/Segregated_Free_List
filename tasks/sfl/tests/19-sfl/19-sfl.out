Segmentation fault (core dumped)
+++++DUMP+++++
Total memory: 384 bytes
Total allocated memory: 32 bytes
Total free memory: 352 bytes
Free blocks: 24
Number of allocated blocks: 4
Number of malloc calls: 5
Number of fragmentations: 0
Number of free calls: 1
Blocks with 8 bytes - 12 free block(s) : 0x9 0x29 0x31 0x39 0x41 0x49 0x51 0x59 0x61 0x69 0x71 0x79
Blocks with 16 bytes - 8 free block(s) : 0x81 0x91 0xa1 0xb1 0xc1 0xd1 0xe1 0xf1
Blocks with 32 bytes - 4 free block(s) : 0x101 0x121 0x141 0x161
Allocated blocks : (0x1 - 8) (0x11 - 8) (0x19 - 8) (0x21 - 8)
-----DUMP-----
